toolchain/sem_ir/inst_namer.cpp

```diff
@@ -491,6 +485,9 @@ auto InstNamer::CollectNamesInBlock(ScopeId scope_id,
         const auto& function_info = sem_ir_.functions().Get(inst.function_id);
         add_inst_name_id(function_info.name_id, ".decl");
         auto function_scope_id = GetScopeFor(inst.function_id);
+        // FIXME pattern and decl are handled separately. Could lead to skew
```

Comment:
My understanding is FIXME means something to be fixed before landing, and TODO: means something to address later? Which one is this?

Reply:
Whoops. That's a note-to-self that I failed to clean up. Fixed now.

(I'm gonna set up a pre-push hook to catch this, because it's not the first time I've done it...)